File: develop-web/src/app/models/dto/createVolumenDto.ts
export class CreateVolumenDto {
  idManga: string;
  nombre: string;
  precio: number;
  cantidad: number;
  isbn: string;


  constructor() {
    this.idManga ='';
    this.nombre = '';
    this.precio = 0;
    this.cantidad = 0;
    this.isbn ='';
  }
}

File: develop-api/src/main/java/com/salesianos/triana/backend/Animangav4/dtos/CreateVolumenDto.java
package com.salesianos.triana.backend.Animangav4.dtos;

import com.salesianos.triana.backend.Animangav4.models.Manga;
import lombok.*;

import java.util.UUID;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
public class CreateVolumenDto {

    private UUID idManga;
    private String nombre;

    private Double precio;

    private String isbn;

    private Integer cantidad;

    private String posterPath;
}
